### PR TITLE
Meeting 97 All Core Devs

### PR DESCRIPTION
**Pull request by alita-moore**

### CODE REVIEW

File: All Core Devs Meetings/Meeting 97.md [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+**97.1.5**: V1: 2315, 2537; V2: 2929; V3: 2718, 2930 -- 2935 (no YOLOs), 2565 -- determined next meeting if going into V3
+
+## Actions Required
+
+| Action Item | Description |
+| ----------- | ----------- |
+
+
+**97.2.1**: Ansger to bring back up CFI for EIP 2938 in two weeks
+
```
Comment (edsonayllon) [inline All Core Devs Meetings/Meeting 97.md:38]: Please have either as a table or as a list. Tables are made separating columns with "|". Lists are made using "- " or "* ".

Comment (alita-moore) [reply All Core Devs Meetings/Meeting 97.md:38]: oh that's a typo, will fix it, thanks!

**Comment by alita-moore** [comment]
first try

**Comment by timbeiko** [comment]
@alita-moore @edsonayllon is this PR still up to date? About to refactor the repo so it would be good to resolve the merge conflicts and merge if so. Apologies for the massive delay!

**Comment by alita-moore** [comment]
@timbeiko yeah it should be!

**Comment by timbeiko** [comment]
@alita-moore got it. I managed to push your changes directly to master. Closing this PR :+1:

**Pull request closed without merging**